compose/project.py

```diff
@@ -13,6 +14,8 @@
 
 log = logging.getLogger(__name__)
 
+VolumeFrom = namedtuple("VolumeFrom", 'source mount_option')
```

Comment:
Ok, will do :wink:.

Moving them in their own file along with related functions should be in another PR I think, right ?